index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+  <main>
+    <div class="container">
+      <header>
+        <div class="logo-container">
+          <P>Emmanuel</P>
+            </div>
+              <div class="menu-btn">
+                <img src="./assets/menu-icon.png" alt="Menu Icon">
+              </div>
```

Comment:
Good job, but make sure to put your menu button into a `nav` semantic tag.

Reply:
Hello @aimalamiri, Thanks for the positive review and the helpful tips. I have worked on all the suggestions you requested, and implemented them in my project in the most recent pull request, have a look